app/helpers/record_helper.rb

```diff
@@ -128,4 +128,19 @@ def restricted_link?
   def guest_and_restricted_link?
     guest? && restricted_link?
   end
+
+  # force http images to use https. If remote server cannot handle https a
+  # broken image is expected
+  def force_https_image_tag(url, alt)
+    image_tag(force_https(url), alt: alt)
```

Comment:
Is there a more 🌈  way to do this? Like with this, we have to remember to use the special function every time we put an image in....is there a way to do it automatically?

Reply:
Since we use two different methods of calling the API, it seemed best to make it a display concern (for now). If we transition everything to the gem, we may get it for free eventually with https://github.com/ebsco/edsapi-ruby/issues/82

Reply:
OK, cool.